core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -1552,7 +1552,7 @@ public static void toXContent(MetaData metaData, XContentBuilder builder, ToXCon
             builder.startObject("meta-data");
 
             builder.field("version", metaData.version());
-            builder.field("uuid", metaData.uuid);
+            builder.field("cluster_uuid", metaData.clusterUUID);
```

Comment:
> Should we mention this in the breaking changes?

this is only used for internal serialization and is not exposed in the api. so I think we are good?

> If we changed it here, should we change it in ClusterStats as well?

Doubting.. I see your point on the other hand it's optional there and doesn't have any uuids. I wonder if it's worth the breaking change.